src/App.js

```diff
@@ -14,6 +12,16 @@ const App = () => {
     const [theme, setTheme] = useState('light');
     const [countries, setCountries] = useState([]);
     const [region, setRegion] = useState('');
+    const [searchTxt, setSearchTxt] = useState('');
+
+    // Search Form Handler
+    const searchHandler = e => {
+        setSearchTxt(e.target.value);
+        setCountries(searchByName(countries, searchTxt));
+    };
```

Comment:
Search is not working as we are constantly reducing the 'countries' state variable as we type in the search bar. Try to delete something from the search bar - no result, as you are passing in filtered list of countries in searchByName function.